docs/content/index.md

```diff
@@ -2,60 +2,153 @@
 layout: doc_page
 ---
 
-# About Druid
+Introduction to Druid
+---------------------
 
-Druid is an open-source analytics data store designed for OLAP queries on timeseries data (trillions of events, petabytes of data). Druid provides cost-effective and always-on real-time data ingestion, arbitrary data exploration, and fast data aggregation.
+Druid is an open source data store designed for [OLAP](http://en.wikipedia.org/wiki/Online_analytical_processing) queries on timeseries data.
```

Comment:
Btw that list of key attributes was just an example. I'm not super attached to those specific ones. I just think it's useful to tell people what we think is special about druid.